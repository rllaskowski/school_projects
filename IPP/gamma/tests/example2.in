## przyk≈Çad gry w trybie wsadowym
B 7 5 3 2
m 1 2 2
m 2 4 2
m 3 3 2
m 1 7 4
m 1 6 4
m 2 0 0
m 1 1 1
m 1 1 2
g 1 3 2
p
b 1
f 1
q 1
b 2
f 2
q 2
b 3
f 3
q 3
