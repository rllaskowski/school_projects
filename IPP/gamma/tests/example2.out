OK 2
1
1
1
0
1
1
0
1
1
......1
.......
.1112..
.......
2......
4
9
0
2
5
1
0
29
1
